Multiplatform/Navigation/Sections/ArtistsView.swift

```diff
@@ -13,13 +13,31 @@ struct ArtistsView: View {
     
     let albumOnly: Bool
     
+    @State var count = 0
+    @State private var working = false
+    @State private var search: String = ""
+    @State private var success = false
     @State private var failed = false
     @State private var artists = [Artist]()
+    @State private var searchTask: Task<Void, Error>?
     
     var body: some View {
         VStack {
-            if !artists.isEmpty {
-                ArtistList(artists: artists)
+            if success {
+                ArtistList(artists: artists, count: count, expand: expand)
+                    .searchable(text: $search, placement: .navigationBarDrawer(displayMode: .automatic), prompt: "search.artists")
+                    .onChange(of: search) {
+                        searchTask?.cancel()
+                        searchTask = Task {
+                            if !working {
```

Comment:
Correct me if I am working, but I think when the user types another character before `fetchArtists` completes `working` will not be set to false and then loading is blocked. Also, is it necessary to check and update the working variable here, the `expand` method does exactly the same?

Reply:
> I think when the user types another character before fetchArtists completes working will not be set to false and then loading is blocked

This is a possibility but probably unlikely to happen because the user has to be that accurate to just wait for 0.5 seconds and type in that character, then this search is canceled. 

The working check is probably redundant here. I'm going to remove it.